src/build/content/server.ts

```diff
@@ -311,47 +311,6 @@ export const copyNextDependencies = async (ctx: PluginContext): Promise<void> =>
   })
 }
 
-export const writeTagsManifest = async (ctx: PluginContext): Promise<void> => {
-  const manifest = await ctx.getPrerenderManifest()
```

Comment:
Looks like we can [delete this](https://github.com/netlify/next-runtime/blob/4b6de32222c22c1d5dda9809d5842e54a45a37f8/tests/utils/helpers.ts#L15-L27) too

Reply:
... Oh, I don't think this is related to your changes, but still dead code

Reply:
Yeah, not really related to this change, but I did remove it now